### PR TITLE
Update WP troubleshooting for new leeway guidance

### PR DESCRIPTION
**Pull request by joshcanhelp**
Updates for v4.0.0 of the plugin

Ref: https://wordpress.org/support/topic/auth0-4-0-breaks-login-invalid-id-token/

### CODE REVIEW

File: articles/cms/wordpress/troubleshoot.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ This means that there is a user in WordPress that has the same email as the one
 1. View the user's profile and scroll down to the bottom
 1. Click **Delete Auth0 Data** and confirm
 
+Keep in mind, if you have 2 user accounts in Auth0 with the same email address, this error message will persist. We recommend looking into [user account linking](/users/concepts/overview-user-account-linking).
```
Comment (lbalmaceda) [inline articles/cms/wordpress/troubleshoot.md]: Should this be a `::: note  :::` ?

**Review by lbalmaceda — DISMISSED** [review]
LGTM.

File: articles/cms/wordpress/troubleshoot.md [hunk 2]
```diff
@@ -126,6 +124,8 @@ add_filter('v_forcelogin_whitelist', 'wp_auth0_forcelogin_whitelist', 10, 1);
 
 ## How can I redirect the users to a specific URL after login?
 
+Please note that all redirects are checked using `wp_safe_redirect()` before being performed. If you're trying to redirect to a domain that is not your main domain, add that domain to the check using the core WordPress `allowed_redirect_hosts` filter ([documentation](https://developer.wordpress.org/reference/hooks/allowed_redirect_hosts/)).
```
Comment (joshcanhelp) [inline articles/cms/wordpress/troubleshoot.md]:
```suggestion
::: note
All redirects are checked using `wp_safe_redirect()` before being performed. If you're trying to redirect to a domain that is not your main domain, add that domain to the check using the core WordPress `allowed_redirect_hosts` filter ([documentation](https://developer.wordpress.org/reference/hooks/allowed_redirect_hosts/)).
:::
```

File: articles/cms/wordpress/troubleshoot.md (hunk 1 above)
Comment (joshcanhelp) [inline articles/cms/wordpress/troubleshoot.md]:
```suggestion
::: note
If you have 2 user accounts in Auth0 with the same email address, this error message will persist. We recommend looking into [user account linking](/users/concepts/overview-user-account-linking).
:::
```

**Comment by joshcanhelp** [comment]
@lbalmaceda - Thanks for the suggestion! Added 2 note call-outs.

**Review by lbalmaceda — APPROVED** [review]

**Pull request merged**